src/pretix/control/views/waitinglist.py

```diff
@@ -125,3 +128,33 @@ def get_sales_estimate(self):
             )
         )
         return qs['s']
+
+
+class EntryDelete(EventPermissionRequiredMixin, DeleteView):
+    model = WaitingListEntry
+    template_name = 'pretixcontrol/waitinglist/delete.html'
+    permission = 'can_change_orders'
+    context_object_name = 'entry'
+
+    def get_object(self, queryset=None) -> WaitingListEntry:
+        try:
+            return self.request.event.waitinglistentries.get(
+                id=self.kwargs['entry'],
```

Comment:
We'll also need to check here if there is no voucher attached, hiding the button alone is not enought. The easies way would be to just add ``voucher__isnull=True`` to this database query here